src/components/DarkMode/globalStyles.js

```diff
@@ -0,0 +1,55 @@
+import { createGlobalStyle } from "styled-components";
```

Comment:
Es muy interesante la solución al cambio de light a dark mode pero en este caso sería más eficiente hacerlo con css normal sin importar la librería de Styled Components. Normalmente se usa una de las 2 cosas, o todo con SCSS o todo CSS in JS como Styled Components. Pero muy bien por tu parte por implementarlo y que funcione 👏🏻